packages/wdio-utils/src/initialiseServices.ts

```diff
@@ -69,7 +75,7 @@ function initialiseServices (services) {
  * @param  {[Any]} service               list of services from config file
  * @return {[service, serviceConfig][]}  formatted list of services
  */
-function sanitizeServiceArray (service) {
+function sanitizeServiceArray (service: WebdriverIO.ServiceEntry): [WebdriverIO.ServiceEntry | WebdriverIO.ServiceClass, WebdriverIO.ServiceOption] {
```

Comment:
After creating the Service interface, use it here.

```suggestion
function sanitizeServiceArray (service: WebdriverIO.ServiceEntry): Service[] {
```